pages/linux/sacctmgr.md

```diff
@@ -0,0 +1,20 @@
+# sacctmgr
+
+> Used to view, setup and manage accounts.
```

Comment:
Perhaps this will read better?
```suggestion
> View, setup, and manage Slurm accounts.
```

Reply:
I initially wrote it without the "used to" but added it in after looking at some page as an example. Can't find that page now, so I was probably mistaken. This is indeed better.